app/routes/contact.tsx

```diff
@@ -0,0 +1,170 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+import type { DataFunctionArgs } from '@remix-run/node'
+import { Form, Link, useActionData, useSearchParams } from '@remix-run/react'
+import {
+  Button,
+  ButtonGroup,
+  Fieldset,
+  Label,
+  Radio,
+  TextInput,
+  Textarea,
+} from '@trussworks/react-uswds'
+import { validate } from 'email-validator'
+import { useState } from 'react'
+
+import { ReCAPTCHA, verifyRecaptcha } from '~/components/ReCAPTCHA'
+import { sendEmail } from '~/lib/email.server'
+import { getFormDataString } from '~/lib/utils'
+import { useEmail, useRecaptchaSiteKey } from '~/root'
+
+export async function action({ request }: DataFunctionArgs) {
+  const data = await request.formData()
+
+  const recaptchaResponse = getFormDataString(data, 'g-recaptcha-response')
+  await verifyRecaptcha(recaptchaResponse)
+
+  const [email, service, subject, body] = [
+    'email',
+    'service',
+    'subject',
+    'body',
+  ].map((key) => {
+    const result = getFormDataString(data, key)
+    if (!result) throw new Response(`${key} is undefined`, { status: 400 })
+    return result
+  })
+
+  let recipient
+  switch (service) {
+    case 'gcn':
+      recipient = 'gcnkafka@lists.nasa.gov'
+      break
+    case 'gcn-classic':
+      recipient = 'gcnclassic@lists.nasa.gov'
+      break
```

Comment:
@jracusin, are these email addresses correct?

Reply:
```suggestion
    case 'gcn':
      recipient = 'gcnkafka@lists.nasa.gov'
      break
    case 'gcn-classic':
      recipient = 'gcn-classic@athena.gsfc.nasa.gov'
      break
```

Reply:
Thanks. Fixed in f87ce9e5ce4593b17fea72b8b948d86e8cd8a1a1.